website/data/docs-nav-data.json

```diff
@@ -751,6 +751,10 @@
             "title": "<code>generate-root</code>",
             "path": "commands/operator/generate-root"
           },
+          {
+            "title": "<code>import</code>",
+            "path": "commands/operator/import"
+          },
```

Comment:
@yhyakuna Hi, there is a new experimental feature released as an Alpha for 1.16 that lets users import secrets from various external secret management services (e.g. GCP Secret Manager) into Vault ENT. Do you know if there is an existing section where this documentation should live? Is it a specific enough use case to be a new top-level navbar item?

Reply:
Sorry I missed this comment in the original review. We don't currently separate docs based on experimental / alpha / beta status because it'll eventually reach GA and moving the docs around creates a subpar reader experience :)